debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleValueConverters.java

```diff
@@ -92,12 +101,17 @@ private SchemaBuilder getNumericSchema(Column column) {
             if (scale <= 0) {
                 int width = column.length() - scale;
 
+                if (scale == 0 && column.length() == 1) {
```

Comment:
Wouldn't this cause issue with something like NUMERIC(1)? Feeling a bit concerned about this one.

Reply:
I reverted this change.